lib/shopping_list.ex

```diff
@@ -17,7 +17,9 @@ defmodule ShoppingList do
     Agent.update(__MODULE__, fn _ -> %{} end )
   end
 
-  def list_msg(content) do
+  def list_msg do
+    content = show
+
     if String.length(content) == 0 do
       "Lista pusta, dodaj cos kumplu :hankey:"
```

Comment:
Don't use if here, there is no need, use case, http://elixir-lang.org/getting-started/case-cond-and-if.html#case it's muc more readable.